docs/regression-tuna.md

```diff
@@ -1,18 +1,23 @@
-# Regression Experiments on Tuna
+## Regression Experiments on Tuna
 
 The machine `tuna.cs.uwaterloo.ca` is a server used internally at Waterloo for development of Anserini. There are a number regression scripts in `src/main/python/` for running end-to-end regression experiments, which include:
 
 + Building the index from scratch.
 + Running all retrieval runs in Anserini documentation.
-+ Verifying results against effectiveness figures reported in the Anserini documentation.
++ Verifying results against effectiveness figures stored in `src/main/resources/regression/all.yaml`.
+
+## Requirements
+
+Python>=2.6 or Python>=3.5
+`pip install pyyaml`
```

Comment:
We can also put this in `requirements.txt` and use `pip install -r requirements.txt` to make it more scalable. But I'm fine with the current way since we only have one Python library to install.